src/index.html

```diff
@@ -11,6 +11,46 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+      action="#" 
+      method="post" 
+      data-qa="big">
+      <div class="big-bar">
+        <label for="big-input">
+          <input
+            id="big-input" 
+            class="big-bar_input input" 
+            type="search" 
+            name="search-big"
+            data-qa="keypress" 
+            placeholder="Try &#34Los Angeles&#34"
+          >
+          <img 
+            class="img-big" 
+            src="images/Search.svg" 
+            alt="#">
```

Comment:
```suggestion
          <img 
            class="img-big" 
            src="images/Search.svg" 
            alt="#"
          >
```
Изображение лупы не надо добавлять тегом img, задвай его как background-image. Оно не контентное, и не должно читаться скринридером